src/Illuminate/Routing/Console/ControllerMakeCommand.php

```diff
@@ -94,7 +94,7 @@ protected function buildClass($name)
         $replace["use {$controllerNamespace}\Controller;\n"] = '';
 
         return str_replace(
-            array_keys($replace), array_values($replace), parent::buildClass($name)
+            array_keys($replace), $replace, parent::buildClass($name)
```

Comment:
are you sure?

Reply:
Yes. Also, tests are green.

Reply:
But this case should be reverted if maintainers feel more comfortable with the older version. So, if you are in doubt, let's just revert it.